StackSplit/SS_splitdiagnosticSetHeader.m

```diff
@@ -44,12 +44,12 @@ function SS_splitdiagnosticSetHeader(axH, phiRC, dtRC, phiSC, dtSC, phiEV, dtEV,
 global thiseq config
 
 axes(axH);
-str11 = sprintf(['%4.0f<%4.0f\\circ <%4.0f']   ,phiRC);
-str21 = sprintf('%3.1f<%3.1fs<%3.1f'           ,dtRC);
-str12 = sprintf(['%4.0f<%4.0f\\circ <%4.0f']   ,phiSC);
-str22 = sprintf('%3.1f<%3.1fs<%3.1f'           ,dtSC);
-str13 = sprintf(['%4.0f<%4.0f\\circ <%4.0f']   ,phiEV);
-str23 = sprintf('%3.1f<%3.1fs<%3.1f'           ,dtEV);
+str11 = sprintf('%4.0f <%4.0f\\circ <%4.0f', phiRC);
+str21 = sprintf('%3.1f < %3.1fs < %3.1f', dtRC);
+str12 = sprintf('%4.0f <%4.0f\\circ <%4.0f', phiSC);
+str22 = sprintf('%3.1f < %3.1fs < %3.1f', dtSC);
+str13 = sprintf('%4.0f <%4.0f\\circ <%4.0f', phiEV);
+str23 = sprintf('%3.1f < %3.1fs < %3.1f', dtEV);
```

Comment:
Have you checked that the header visually still looks fine when removing the whitespaces here?

Reply:
At the moment I am unfortunately not completely sure which white spaces you mean. The white spaces I removed are all outside of the text string. The added white spaces should add consistently a white spaces before and after the `<` signs.

_Diagnostic plot with main branch_:
![StackSplit_Multi_result_SIMW_main](https://user-images.githubusercontent.com/94163266/213656054-5163ed0d-baf4-45e1-bf97-0c99f2f6ae7b.png)

_Diagnostic plot with changes in this PR_:
![StackSplit_Multi_result_SIMW_PR11](https://user-images.githubusercontent.com/94163266/213656134-81b30cd8-7399-4f7f-bd7a-43d81bcba42b.png)

Reply:
Ok fine!